streams/src/main/java/org/apache/kafka/streams/state/internals/CachingKeyValueStore.java

```diff
@@ -113,37 +99,92 @@ private void putAndMaybeForward(final ThreadCache.DirtyEntry entry,
         }
     }
 
-    public void setFlushListener(final CacheFlushListener<K, V> flushListener,
-                                 final boolean sendOldValues) {
-
+    public boolean setFlushListener(final CacheFlushListener<byte[], byte[]> flushListener,
+                                    final boolean sendOldValues) {
         this.flushListener = flushListener;
         this.sendOldValues = sendOldValues;
+
+        return true;
     }
 
     @Override
-    public void flush() {
+    public void put(final Bytes key,
+                    final byte[] value) {
```

Comment:
Just to confirm: this line and below do not have any logical changes just due to git diff messing the line numbers right?

Reply:
Yes.